doc/design/regularization.md

```diff
@@ -18,52 +18,21 @@ The most commonly used norm penalties are the L2 norm penalty and the L1 norm pe
 ##### L1 Regularization
 <img src="./images/l1_regularization.png" align="center"/><br/>
 
-A much more detailed mathematical background of reguilarization can be found [here](http://www.deeplearningbook.org/contents/regularization.html).
+A much more detailed mathematical background of regularization can be found [here](http://www.deeplearningbook.org/contents/regularization.html).
 
+## Regularization SUrvey
```

Comment:
SUrvey => Survey

Reply:
Fixed in 8c30c102669c8405b0b034ed72e1c27eb810889f